_includes/responsive-image.html

```diff
@@ -0,0 +1,58 @@
+{% comment %}
+<!--
+  Responsive Image Include
+
+  This partial is used with the responsive image plugin.  In the _config file,
+  this template is used as the default for responsive images to configure a
+  srcset value for images to ensure the best option for an image is used for a
+  specific user's browser.
+
+  @param {string} "path" - Required. Path to image file relative to the root
+  @param {string} "alt" - Required. Alt tag for image
+  @param {boolean} "img_link" - Optional, default is false. If img should have
+    popover link
```

Comment:
Can we use other terms to explain this? I don't recognize the term "popover link."